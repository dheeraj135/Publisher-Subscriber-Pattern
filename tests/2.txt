T topic8 KXZTgLEaDovKIoq
S topic4 0
U topic6 3
T topic5 IsdKjMLOEqVQQHE
S topic6 0
T topic4 qMgDS20dBNCspdE
U topic8 1
U topic5 4
S topic2 4
T topic1 uJVv5FuSmui1gAp
U topic1 4
U topic5 4
U topic0 3
U topic7 4
T topic2 0aKjik82i53AgSP
T topic1 8qcBFVLViHlD0LR
S topic1 2
U topic8 2
U topic2 4
U topic7 3
U topic1 0
T topic8 uDsaJKYBEZJID8p
T topic3 jKrQMk2liMuCAHr
U topic5 3
U topic3 4
T topic4 FBOuuLT9SkiUyvg
T topic7 3fTIG7qyxScLMUr
U topic8 2
S topic1 2
T topic8 lZiacytWBIjO0tt
U topic8 0
S topic6 2
S topic4 0
U topic5 3
T topic7 Ole3M5Hf4I5X5q8
T topic8 nthC2xTaEXidYYX
U topic7 1
S topic8 1
T topic9 c6Y6hVpIyudNpDB
U topic7 4
S topic8 1
T topic8 hDd5e0AFI6eWN8z
T topic2 T8GcWkh2IACcejv
T topic6 TqKjZAgWDmFeVDS
T topic4 I2y6dxLhPVK41LQ
T topic1 AtpOIs7tmRWByAq
S topic3 1
T topic7 CbpONYGMARb7qbf
S topic6 1
U topic2 3
U topic5 4
T topic6 WP5zv561m5aKARY
U topic5 4
S topic3 2
S topic3 3
S topic4 4
U topic4 1
U topic4 4
S topic1 0
S topic5 4
U topic5 4
T topic1 zqDiCkHIvLLXQ5K
T topic4 a4pYlthCNiIc4fn
U topic2 4
S topic5 2
T topic0 PuzN59vYWB9Tiee
S topic0 0
U topic9 2
U topic8 1
U topic9 0
S topic3 4
S topic0 1
S topic2 4
U topic2 3
S topic2 4
U topic4 3
T topic2 eUfGukOX0TIYscr
S topic6 2
S topic2 4
T topic2 6Jq6HP1vhVD3yWx
T topic4 ZKn0R2eH7xXUdeA
T topic5 jgCvVVK4b0BFlC0
T topic9 XZhRlPODDhABcPQ
S topic8 3
T topic0 ieaREpm9z9g39qi
S topic3 1
S topic6 0
T topic4 ajszsMNUXgcMTIR
T topic9 N26HyWDpkgrUHoJ
T topic5 6TA20XI5nQ1fIs9
U topic8 0
S topic5 1
S topic7 2
U topic6 0
T topic8 8uapdqtV2rRs4Qi
T topic9 xj8sh6Nk3MbSCOi
S topic1 3
T topic5 N2GTO37mttWyAml
S topic2 1
S topic8 2
S topic8 3
U topic7 0
S topic3 1
S topic5 4
S topic9 0
U topic7 3
T topic6 JvJy1jrCEatqbHB
S topic9 3
T topic5 FE0nGPKeBB3hPXe
T topic1 YPzIYiJV8ZjvpLh
T topic6 uG0zeLop2dv7C8N
S topic9 3
U topic0 4
T topic9 IlT8GLvOzuUfSji
T topic6 kkkcBLBCtrWuJYk
U topic9 1
T topic0 aUl8IL9VgCfGtDp
T topic9 7CAyGcdRKpO9f5f
S topic9 2
S topic2 3
S topic7 3
T topic7 zCqKUJN7tVRabAP
U topic0 2
T topic9 m30XZ6aOcHA8x0Y
T topic1 cRisIlbC8Ltlzq4
T topic8 1nRbHvD3MJrRc0n
T topic0 299DnpveeiEXp7O
U topic3 4
T topic0 OC4DUQXJTo5avoa
S topic2 0
U topic3 3
T topic2 uef1t0TsrU6iynq
S topic3 2
U topic5 3
U topic9 0
U topic2 1
U topic5 3
U topic5 4
S topic0 2
S topic7 2
S topic3 1
S topic0 2
S topic8 1
S topic1 4
U topic1 4
T topic8 vFjHLMmumcrsQO0
S topic5 4
S topic7 1
S topic3 4
T topic2 WWxTepjXapDiQj6
S topic5 4
T topic4 ea8M1E5MX05QyzL
T topic6 ceTBZ69l5rnoFNJ
S topic6 2
U topic2 1
T topic9 DyqJhqKMkHkN7mp
U topic6 1
T topic6 REk84PpWUVhbmCt
U topic3 4
U topic2 2
U topic9 4
U topic7 2
U topic9 4
S topic3 1
U topic5 3
T topic1 dNRhbLBWWcs5roI
U topic7 3
U topic0 4
T topic6 Q6pFoljc1kCtKuJ
T topic4 sOgyTlIUTeUUZQ5
S topic2 0
U topic5 1
U topic5 3
T topic9 4WKmGWXIJxVNdUV
U topic5 0
U topic9 2
S topic0 1
S topic1 0
U topic1 4
T topic2 C9KC8Y18jqdHc4T
U topic3 0
S topic4 1
T topic0 aMRHZ11kbOKGyyu
U topic1 3
T topic2 lA64dWjPriC9fhh
U topic4 2
U topic7 0
T topic6 BVSNIQtiEzvTPyu
S topic3 2
S topic4 4
U topic8 2
S topic3 2
S topic4 3
T topic6 Qb3IxnrVM2YX4AS
U topic1 3
S topic4 4
T topic0 WvW6Qx4CGIHto67
U topic8 1
T topic0 cqpuuzvGJ5iyUbM
T topic3 hZ7harR4tdBYEEq
S topic4 4
U topic0 4
S topic3 2
T topic6 KAripGL8PrsW4gg
T topic3 GsEJFqg2y2ARSfT
S topic6 0
S topic4 2
S topic1 4
U topic4 3
S topic1 2
T topic2 8F19xTi39NWOFGB
T topic0 jdIMCFVXGaHEkPi
T topic0 DEG8Up2ujaa7HeJ
S topic7 4
S topic1 4
S topic5 3
S topic1 2
S topic9 4
U topic0 3
T topic3 jNvOjN1v9V9XJ6v
S topic3 4
U topic9 2
S topic4 4
S topic5 4
S topic1 0
S topic1 1
U topic0 1
U topic4 2
S topic8 1
S topic2 3
U topic0 2
U topic5 3
T topic8 TqV3QMfgeUVdt4e
U topic7 2
U topic7 4
U topic3 3
T topic4 GvVjaSPLQU2EL3u
T topic8 5o2vAALpgXqGCMr
U topic2 4
U topic8 4
U topic2 3
U topic4 1
U topic7 2
T topic7 nhXAce4dqdXyA5g
U topic5 3
T topic5 FgFIjD0BN84pOA0
U topic1 0
U topic3 4
U topic9 4
U topic1 4
U topic6 2
U topic8 2
U topic2 1
U topic9 2
T topic6 EFbvhKfRF3e70zf
S topic1 0
U topic2 1
U topic8 3
T topic0 zCJZjSZ9WiHSEMi
S topic1 2
S topic5 0
S topic6 1
U topic7 4
U topic6 0
T topic4 k77QTj9SESCMptu
S topic4 0
S topic7 1
U topic8 4
S topic7 0
T topic2 1hINzQmr2rjMWaW
S topic6 1
T topic3 7SCXn2kMnVIHzOI
S topic3 3
S topic5 4
T topic0 5kSjzQvWqRIFbqc
S topic9 1
U topic5 4
U topic9 2
U topic0 2
T topic0 kltYdedUcPva224
U topic9 4
S topic1 4
U topic4 1
U topic0 2
T topic8 N3xk1e4fXolou3X
S topic0 0
S topic3 2
T topic2 AgHedgDvUnBHlSl
U topic2 1
S topic9 3
T topic1 SV7ztdEhIpCyeng
U topic7 0
U topic7 0
S topic3 0
U topic3 4
S topic7 4
S topic7 4
U topic4 4
S topic6 4
T topic4 jUj1dWtBJrpBYHQ
S topic5 1
T topic5 TsuBzLcJNptv2vN
S topic3 3
S topic6 1
S topic9 3
S topic8 4
U topic7 3
S topic3 3
U topic3 4
S topic3 0
S topic8 3
U topic8 4
U topic5 4
U topic2 1
T topic1 DJdTEPjQWeGnTVf
S topic6 4
S topic4 3
U topic1 3
T topic5 67SEQXc0k3jUZxk
T topic6 SWcZEoS3YOWkk6g
T topic6 sepVq8clF9RrRdY
S topic3 3
U topic6 0
T topic0 BbNCmAEHcWWQFDz
S topic9 2
U topic9 1
S topic9 1
U topic9 0
T topic7 VfAfM5MVYPxztbf
T topic9 6Ux7cQMxVnCCsVJ
U topic6 4
T topic2 r2ZtQXJsnOSA3i1
S topic4 1
T topic2 E8VXF1k6YQr7Wo1
U topic5 4
T topic0 bUXfOW5B4Y9IfJ4
S topic1 1
T topic6 xjOMibQRtV66rAl
S topic7 1
S topic0 0
S topic2 2
U topic7 2
U topic8 4
U topic5 1
T topic0 spmezG2KUibXDbJ
T topic5 leg5NbZVQF2FSqv
U topic3 4
T topic8 MTy5jdGkCVaLtMv
U topic7 3
T topic0 dZFpimRpUsPRKAK
S topic4 0
S topic1 2
U topic1 2
U topic7 0
T topic9 C3ql4L4nHpWKX6Y
S topic2 0
S topic5 0
U topic6 3
U topic3 2
S topic6 2
T topic6 Dxi2zqpVvfz0dK9
T topic8 iqvWPps7P4dny2H
S topic1 3
S topic8 3
S topic0 1
T topic1 3pFnGpnWQ1WYEPJ
S topic9 4
U topic8 4
T topic5 csdBLsI1J4tru6P
S topic8 3
U topic6 0
T topic9 Zsioj5WUQGTyBxQ
T topic5 HHKvtQFrB0aV9Md
U topic4 3
T topic2 o6XM6PTvNlkV18c
S topic9 4
T topic2 M0ABK0WSjIdXE1a
U topic8 0
T topic8 YoIWKhgEp64Oo5X
U topic7 0
T topic6 OPesxTY5D33IhjI
U topic3 3
S topic3 0
U topic4 0
S topic7 3
T topic1 zanH9GZ1bOH06Sv
S topic7 3
T topic1 M8GiMFS5k1b17vq
U topic4 4
U topic6 4
S topic0 1
T topic6 MfGuLLIfVMhqIhb
T topic1 k0Ts0TPXG3SiQHA
S topic7 4
T topic9 TxWjLUB6Ge9FFia
U topic3 0
S topic3 4
U topic2 0
U topic9 2
T topic2 vrZn9brcGNky0Po
S topic6 3
T topic5 ypL84b1kEYt811W
S topic0 2
S topic9 0
T topic8 AltNNOHbAk64C13
U topic4 1
T topic3 tBY1UbnDjaGCBYU
S topic2 0
S topic7 2
T topic0 DoKdkjeP6mGfaEJ
U topic5 4
U topic1 4
S topic8 4
T topic8 J1RG2PQ5hAgBxIM
T topic3 20VsOKLKUlykJro
S topic1 4
T topic9 iO1yghvcXR1Ovdl
S topic4 2
U topic7 2
T topic6 aAZaOaKtCioXepH
U topic8 0
T topic1 T3MCQa2qZE3HB9k
U topic4 1
T topic4 0b228cndeqCGpm8
S topic8 0
S topic8 2
S topic1 2
S topic6 0
U topic3 1
S topic9 4
T topic8 Q57LanFSYYn9RER
U topic7 3
U topic7 4
S topic8 0
T topic5 GtoIJJWca1F9tr8
U topic9 2
U topic1 3
U topic5 4
U topic0 4
U topic4 3
S topic7 2
T topic1 9T6DHmx4nOb4HW3
U topic9 4
T topic3 FTmcJgxSWfq6c27
T topic0 rOM0PmP1JIOi23U
S topic0 3
T topic8 gqLCTEP7kXGrSLm
S topic6 3
S topic8 3
U topic2 0
T topic5 YeJDPizXfyvc45r
S topic0 1
S topic7 3
S topic6 0
T topic6 AeIgCEDmY4xo2Ys
S topic3 1
T topic7 TK74tSxB35s1V8u
S topic1 0
U topic2 1
S topic2 4
S topic8 4
S topic2 3
S topic0 0
T topic2 TpQBr1cs69Z7juR
S topic9 4
T topic8 zxfQ4spfrFHj3ER
T topic1 YI5Ifj2ePLpaHBa
U topic7 2
S topic3 1
S topic0 3
U topic9 2
U topic7 3
S topic8 3
U topic5 1
U topic9 0
T topic6 JGfgHp87duxrd5l
U topic5 3
T topic3 NXzRLYe5gKm38du
T topic6 ltbKdeVPWbqelrD
S topic3 3
S topic1 2
T topic9 0mqJJcWWkbh3MAm
U topic1 0
T topic2 jimFir1ikPiYtaA
T topic1 qrkZDOcX7WsZ62M
U topic9 3
S topic0 2
U topic0 4
U topic5 0
T topic7 n4yErcUI2N8th4T
U topic0 1
S topic0 1
S topic8 1
U topic7 0
T topic8 jLo6LfvISj2FDBa
T topic4 PUJB99kkkV6NMnx
S topic1 3
U topic2 2
T topic7 yWKeLGr6ix9MAdD
T topic7 QkZHGvO7VB2XvSz
S topic1 4
T topic4 5WynmyNQjjCcDH9
T topic2 kdA0Xc0HcpOUdo8
T topic6 UxSl4u4Ji3uvbI5
U topic2 3
S topic9 1
U topic4 2
U topic5 4
S topic0 0
U topic2 3
S topic6 1
U topic4 1
U topic9 1
S topic6 0
U topic7 0
T topic8 49Uq1uJLkCVE6IQ
S topic7 4
S topic0 2
S topic0 0
S topic6 0
U topic1 3
T topic8 bVbDFNYo5fz09KB
T topic5 OzHEL6xNHUnXsyn
T topic9 xUouR3cJiCDSBlv
T topic3 ZsQTkuENAnUHNlB
T topic6 D3EQfbyWAHNOGYs
S topic3 4
U topic7 3
U topic5 3
S topic6 2
S topic7 1
S topic5 2
T topic9 B59IOinyuOXrp8N
S topic8 1
S topic2 1
U topic7 3
U topic9 1
S topic1 2
U topic5 3
U topic9 1
T topic7 yT3N2kr9a9W5FSb
U topic1 3
U topic3 2
T topic0 06mebdNlC6Zzeg2
S topic8 3
T topic9 VtDCQpIeLP8AkjG
U topic5 0
U topic3 4
T topic9 boeaPVvGtajKe7t
U topic6 2
T topic0 T8v6DK8R3QJDIn4
S topic2 4
U topic6 1
S topic7 0
T topic1 8ndtNjkKCE8NujS
T topic2 5e1VfHnk2pClORC
S topic0 2
S topic8 3
S topic1 4
S topic7 0
S topic4 3
T topic4 G49IgmTyva1VzTy
U topic7 1
T topic5 agFHdOV0x2jl7zD
T topic3 UWsENGvShXk6p9B
S topic4 1
U topic5 1
T topic3 HZF2Xfg5tvS1FDB
U topic2 2
U topic9 3
S topic1 2
S topic6 0
S topic7 1
T topic8 XAHgHmivj65HBE2
T topic3 GEbxVPB7SP3VUW3
S topic0 0
U topic6 2
T topic8 XVPzi2Hl7TNZH5F
S topic6 2
S topic9 4
T topic3 4nBuroSkBlz78HG
T topic7 jr2zN30bQKtrAd7
U topic7 1
T topic4 9n919ysVqxj2Po8
S topic0 2
S topic4 4
U topic9 2
U topic0 4
U topic1 0
U topic5 2
U topic9 2
U topic8 0
U topic3 1
T topic3 XR8y8qyrm1micDP
U topic6 2
T topic9 kmgiFQk2Uh6dXOx
S topic0 1
S topic1 3
U topic1 1
U topic6 2
S topic9 3
U topic6 2
S topic6 3
U topic8 0
S topic2 1
T topic2 93CZC7PgRAJnysS
T topic9 HD5k558mHMsDDM0
S topic3 0
U topic1 4
S topic4 0
T topic7 sETxI9ie30IvPY6
S topic2 4
U topic0 4
S topic0 4
T topic4 y3Yp3FLgUutS7l4
U topic0 2
U topic3 1
T topic5 8a42e45Hq1mvTDR
U topic4 0
U topic9 0
T topic4 1O82sQu5naklWVV
T topic8 D28n1LmteksvtCF
U topic1 3
U topic8 2
U topic1 0
S topic8 2
S topic3 4
T topic8 Tt8mRxBOvSF5opZ
T topic4 lUidVbFkIGovUN2
T topic3 dKorsCiRcKjBRMp
U topic1 4
T topic3 1Y1dR4OiRyQEYTq
S topic4 0
U topic1 1
T topic7 pGUBd65OgeUaW80
U topic1 1
U topic5 1
U topic0 3
U topic4 0
S topic0 2
U topic6 0
U topic0 3
T topic2 63TR3BPScr5dFG7
S topic3 3
T topic6 e70eBPqrzmcoCs5
S topic2 0
S topic4 1
S topic5 2
T topic9 Tl4F8u02d0XryuO
U topic6 1
S topic2 2
S topic3 2
T topic5 RCq9dlvXpiUFSpg
S topic5 3
S topic8 0
S topic9 4
U topic8 4
U topic6 3
U topic6 2
S topic4 2
T topic2 Wai0YJlf7FsoOxI
T topic5 bkO2OxQanqoGHfm
U topic9 0
S topic7 3
U topic7 3
T topic8 PW6LDWMF2V40mib
T topic7 XZoBJ2Mcat1Ofxm
U topic2 0
U topic5 0
T topic6 biU3dbUa0QhHHlu
T topic2 kbPHztu4xnJSVfA
U topic2 2
U topic5 3
U topic9 3
S topic7 0
U topic0 4
U topic7 4
T topic9 FXGhk18ka7sRBD9
U topic1 0
U topic0 1
U topic2 2
S topic2 3
T topic7 pHFS7jsLjac74kh
S topic6 1
U topic3 3
S topic4 2
U topic2 4
T topic0 gZTX2zYVfR0qD37
T topic7 bhkUzM8UKOl998Z
T topic4 1iHGzoH31ASthQY
T topic5 h24eLdbFYQvuNWx
S topic3 0
T topic2 RWhvU97eHyiSKB9
S topic3 4
U topic9 1
U topic0 2
T topic1 nLXamfocvtpNgoZ
S topic3 0
S topic6 3
S topic4 0
S topic4 1
T topic5 a7OzUCjn72j5QdO
U topic1 0
S topic9 4
T topic6 ceOTfWSRxsDM1T4
T topic6 649l8JJo83oOTMM
T topic0 qjXdkcItI86qd0o
U topic9 2
T topic2 iY3ARFsziZg5v71
T topic8 GqH4xyvSU57pLqn
U topic9 0
S topic3 1
U topic5 4
T topic2 tQMCivq4VV1kEvn
T topic1 AX2zSWiox6DOE5o
S topic7 1
T topic6 YzhaOcNzHXxZmRD
U topic3 0
U topic3 3
S topic3 2
T topic9 shhaJ61L9Zu6Yon
U topic8 0
S topic5 3
U topic1 2
T topic9 HJnQxNFFHy5bLdn
U topic0 1
U topic5 4
T topic6 xNu0Th0Rfgz1iiY
S topic7 4
T topic7 rHiixpUZ0YaeNkj
S topic9 0
U topic9 2
T topic3 pA1SbfCgoIFr3tp
T topic5 IkOX8DVmqAtWah3
T topic2 mebNc1PpGIXSH4J
S topic0 2
U topic7 4
T topic9 ycKtdHnBZIuLN1J
U topic6 2
S topic5 0
U topic4 1
S topic0 2
S topic1 2
U topic7 2
S topic0 2
T topic6 tRmhzERfApUyIyD
T topic8 vcWg4LdXlbIz23Q
T topic6 yggLFmUTHeKVCUm
U topic3 2
T topic9 IqIDncV5jSMolK8
U topic7 0
S topic6 4
T topic3 Pe6nq24z1ltcgPL
T topic5 ghfIAbhvnymysmy
U topic2 0
U topic8 4
U topic0 2
U topic4 1
U topic3 3
S topic6 2
U topic0 1
S topic7 3
U topic8 2
S topic0 1
T topic6 pMJX9F53zZDSDeD
S topic7 3
S topic7 4
S topic4 3
S topic0 0
U topic2 0
S topic2 3
T topic7 RhPDh3qzQOgAd59
T topic1 SDcNBn2T0R1jTzR
U topic6 2
S topic6 1
T topic2 KOzL9ULzTUedV8H
U topic9 2
S topic6 4
S topic3 2
U topic2 2
S topic7 2
S topic6 3
T topic4 2AX9mcuu3t07jbr
S topic9 2
S topic1 3
T topic4 p89Eti4EfCRU5by
U topic6 2
U topic1 4
T topic9 eEVSTJnF8hW8JaA
S topic0 4
U topic5 3
S topic1 1
U topic3 1
U topic1 1
U topic5 1
U topic1 0
T topic8 8p1ZVmzENh8VNah
T topic2 gyMn58i6ChjyqLE
T topic0 pUFWCULoUyjvSNk
T topic2 PWpgtnQiGqmCTNi
T topic5 EfWZuGk58oBq2Cz
T topic1 H2nCq0e7MiVO7UN
S topic8 1
S topic0 4
U topic4 4
T topic2 tqSDrkcPOpZAZeI
S topic9 2
U topic5 1
S topic9 0
S topic5 0
S topic0 0
S topic2 1
T topic4 cU2pTQA5zgnirb5
S topic8 1
T topic3 DkxfY6hv4cRHFf6
U topic2 1
T topic1 ArfbbfeVCrp4Sqa
U topic4 3
S topic7 3
U topic8 1
U topic5 2
S topic5 0
U topic7 0
T topic6 jSdTVnShSK3Ez5e
U topic9 4
T topic1 Dh6hcCL9NOUfb2i
T topic1 z57A8CMB3ihGeGz
T topic3 RWSa8Fr0cjiqXOY
S topic6 1
U topic6 2
S topic3 0
T topic7 NVrRfZ4ByUWedxh
T topic5 nynSnRAn7yuCVc0
S topic3 3
U topic3 2
S topic9 0
U topic3 2
T topic1 U9hJ4OT6ubp2EYX
S topic1 0
U topic3 1
T topic7 loz1h3pnZMUSfQH
S topic2 0
U topic9 3
U topic4 4
S topic5 1
U topic6 0
S topic3 1
U topic2 4
S topic4 0
U topic5 4
S topic7 4
S topic7 0
U topic8 0
S topic3 2
T topic6 J1847nf0NUnx55N
T topic1 4TAh8p0zLiX1puZ
T topic3 f4NTE6ihuNCijtK
U topic9 1
T topic9 mM65VuLft6WKPPu
T topic4 jFrOFT4V4T7nrtC
S topic3 4
U topic6 0
S topic3 1
T topic8 DdE427k4LK6Hnxu
T topic4 i2sqh6yiN33jAQJ
U topic5 3
S topic5 4
U topic8 3
T topic3 jWDux37drifCXF3
S topic9 0
T topic1 Dx8SrJgTkqgyLf6
U topic5 0
U topic6 2
U topic6 1
U topic0 0
S topic1 4
S topic0 1
S topic0 1
U topic4 3
S topic6 0
S topic9 2
S topic7 1
U topic7 1
S topic9 1
U topic6 2
U topic6 0
T topic5 e6d4Op5LGI2NzYi
T topic6 QeNCT2ry6yCJR83
S topic8 0
S topic2 0
U topic6 2
S topic5 4
T topic4 Yx4X2PZEQ9zvRRO
U topic6 0
U topic0 1
U topic4 2
U topic8 0
S topic9 4
S topic8 1
S topic9 1
U topic9 0
S topic1 1
S topic3 4
S topic3 2
T topic8 Li6zA6KaVWmHUjI
S topic4 3
S topic1 3
S topic9 2
U topic5 4
S topic1 2
S topic4 1
T topic3 mhLsWOnpBTQzs7N
T topic5 2UcY3mu1ihrPIkb
U topic0 0
S topic6 4
S topic0 0
S topic0 4
S topic7 3
U topic5 0
S topic7 4
S topic7 2
S topic1 2
U topic7 3
U topic1 0
T topic1 BYefpxvd8VWn8eM
T topic7 E3XCzq49m2NEO33
U topic5 0
U topic4 0
T topic6 bzheRIe4DL93Oyo
U topic9 2
S topic2 0
S topic6 3
S topic1 3
U topic8 1
T topic7 mufPkWdlpt9Wq4k
U topic6 0
U topic1 2
U topic6 0
S topic0 0
U topic8 2
U topic5 3
U topic9 4
U topic1 3
S topic1 2
T topic6 C8nnueCYXYOrk8o
S topic9 2
T topic8 0UevK4qRVGQYLhZ
T topic7 iuhmcmgPhQfMSaO
U topic3 3
U topic3 0
U topic4 4
T topic7 rdNAB5pzt9kFhDi
S topic5 3
U topic1 2
U topic1 2
S topic6 4
S topic7 2
T topic9 bQNKiNjde8DMd0S
U topic8 3
S topic3 3
T topic3 AuEBNuoCfgExLZN
T topic9 ntxVCBsr2gx6dfg
T topic4 RjaRD9vvFxdP06h
S topic0 2
S topic4 3
T topic3 kPSkjhEZrT3u32k
T topic9 rfUxtPqHXcFGUrT
T topic2 YU2Jz8jy6SeXbRQ
U topic9 0
T topic0 Hl8LomLbAeXYkFr
T topic5 Wu5b6oWxT0b10Ty
S topic2 2
T topic4 GNp9OMr8KYAovtX
S topic7 3
S topic9 2
S topic5 4
U topic5 0
U topic6 2
T topic9 Hz3JvXpla5by487
U topic1 0
T topic4 vUrCD1aqKNGgCXE
S topic7 0
U topic1 1
S topic6 2
U topic0 3
T topic7 n7x1Z9JocSPQvd4
S topic3 2
U topic4 2
T topic6 epSkO0tyHeljGcH
S topic5 4
T topic2 Gd5MIahtghNC0nT
U topic2 2
U topic0 2
S topic4 1
U topic2 0
S topic5 1
U topic0 1
U topic8 3
T topic5 La9J0ZO5OXzjPrh
T topic0 eNng3L4SR1Gtvby
