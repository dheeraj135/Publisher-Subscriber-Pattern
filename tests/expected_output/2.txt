gCFBeEjbgSncS0k
TvE77YfY4brMbUk
M2e4d3e2kZrWFRx
9kU34JeaMycgu04
jnPGHBxzuk487IO
9NOVFuVpjeieu7f
rmJ3H5MA2db800c
3mGECR6xdj6d7Lu
AajJ6xd8d4WmsvC
23eKqe2877Cykgh
Gsdbsvj0z2X8P5b
9X0trj6mNSCjEl4
eHxcLalOvIjIItS
nFbBYzLYcryHMzW
zFiR95FMsxnuasK
6e7C117WX7832bX
UqGjat6mjsneCs9
2jMlJFEhZIR0JDt
IsCSlBKvLYvBtej
3gU7I5khQHcjc38
eaHHyP22FPJ1UOm
tvAFM5p12XfyXzp
ena2xGm79CIHiq6
gCAzGDmSxNKuAMF
Cgiy9CDqLfyxSFc
1T5dDlrbyWdBmm0
d88ylpzPOoqtJI7
h4XLmb5ET7ohDPz
AGIRVByMoV2etdd
1hnvsO2GEvePzAy
0avUGqdPjhp3tuY
